doc/api.rst

```diff
@@ -538,6 +539,25 @@ Runner class
 
       See the :ref:`bench_func() example <bench_func_example>`.
 
+   .. method:: bench_async_func(name, func, \*args, inner_loops=None, metadata=None)
+
+      Benchmark the function ``await func(*args)`` in asyncio event loop.
+
+      *name* is the benchmark name, it must be unique in the same script.
+
+      The *inner_loops* parameter is used to normalize timing per loop
+      iteration.
+
+      To call ``func()`` with keyword arguments, use ``functools.partial``.
+
+      Return a :class:`Benchmark` instance.
+
+      .. note::
+         If the ``func()`` leaves any tasks, it affects to next iteration.
+         So you should care about task leak for reliable benchmark.
```

Comment:
This note is now outdated, since a new event loop is created at each iteration, no?